aptos-move/framework/aptos-framework/sources/resource_account.move

```diff
@@ -42,7 +42,8 @@ module aptos_framework::resource_account {
     use aptos_framework::account;
     use aptos_std::simple_map::{Self, SimpleMap};
 
-    const ECONTAINER_NOT_PUBLISHED: u64 = 0;
+    /// Container resource is expected to exist on the specified account.
```

Comment:
This is another one of the inconsistent not found messages.

Maybe just `Container resource not found in account`?  or just `Container resource not found`

Reply:
Updated